src/modules/volumes/volume-list/volume-list.js

```diff
@@ -164,48 +220,25 @@
             _applyFilters(filters);
         }
 
-        function startTimer() {
-            volumeTimer = $interval(function() {
-                init();
-            }, 1000 * config.refreshIntervalTime, 1);
-        }
-
-        /* Trigger this function when we have cluster data */
-        $scope.$on("GotClusterData", function(event, data) {
-            /* Forward to home view if we don't have any cluster */
-            if ($rootScope.clusterData === null || $rootScope.clusterData.clusters.length === 0) {
-                $state.go("clusters");
-            } else {
-                init();
+        function _compareFn(item1, item2) {
+            var compValue = 0;
```

Comment:
can avoid If else by doing the below.

var sortId = vm.sortConfig.currentField.id;
compValue = item1["sortId"].localeCompare(item2[sortId]);

Reply:
@a2batic Take it as a refactoring for other views. If it improves this we can work with other views and maintain that consistency